examples/nlp/active_learning_review_classification.py

```diff
@@ -281,15 +282,19 @@ def train_full_model(full_train_dataset, val_dataset, test_dataset):
         ],
     )
 
-    # We will save the best model at every epoch and load the best one for evaluation on the test set
+    # We will save the weights of the best model at every epoch and load the best ones for evaluation on the test set
+    checkpoint_file = "FullModelCheckpoint.weights.h5"
```

Comment:
Saving weights because of [this issue](https://github.com/keras-team/keras/issues/18913).